hack/test-go.sh

```diff
@@ -32,11 +33,16 @@ find_test_dirs() {
     \) -name '*_test.go' -print0 | xargs -0n1 dirname | sort -u | xargs -n1 printf "${OS_GO_PACKAGE}/%s\n"
 }
 
+special_upstream_test_dirs() {
+  echo "${OS_GO_PACKAGE}/${KUBE_GODEP_PATH}/api"  
+}
+
 find_upstream_test_dirs() {
   cd "${OS_ROOT}"
   find ./Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes -not \( \
       \( \
-        -wholename './Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/runtime' \
+        -wholename "${KUBE_GODEP_PATH}/api" \
+        -wholename "${KUBE_GODEP_PATH}/runtime" \
```

Comment:
Runtime is excluded because one of @smarterclayton's patches doesn't have unit tests, nor does the upstream PR.  One of my last conscious thoughts last night was that I should make a comment explaining it / open an issue.